.travis.yml

```diff
@@ -1,8 +1,20 @@
-language: node_js
 sudo: false
+
+git:
+  depth: 1
+
+branches:
+  only:
+    - master
```

Comment:
I think it reduces load on the CI by only [running on specific branches](https://docs.travis-ci.com/user/customizing-the-build/#Safelisting-or-blocklisting-branches) e.g. if you create a branch and push it up, the CI won't run. However, it looks like [PRs are treated differently](https://docs.travis-ci.com/user/pull-requests) and when a PR is created the CI runs by merging into master, which explains why the test I just did correctly failed.

Shall I merge?